app/assets/javascripts/templates/admin/schedule_dialog.html.haml

```diff
@@ -1,24 +1,24 @@
 #schedule-dialog
   .text-normal.margin-bottom-30.text-center
-    %span{ ng: { hide: 'schedule.id' } }
+    %span{ "ng-hide" => 'schedule.id' }
       {{ 'js.admin.order_cycles.schedules.adding_a_new_schedule' | t }}
-    %span{ ng: { show: 'schedule.id' } }
+    %span{ "ng-show" => 'schedule.id' }
       {{ 'js.admin.order_cycles.schedules.updating_a_schedule' | t }}
 
-  %form{ name: 'schedule_form', novalidate: true, ng: { submit: "submit()" }}
+  %form{ "name" => 'schedule_form', "novalidate" => true, "ng-submit" => "submit()" }
 
     .text-center.margin-bottom-20
-      %input.fullwidth{ type: 'text', name: 'name', required: true, placeholder: "{{ 'js.admin.order_cycles.schedules.schedule_name_placeholder' | t }}", ng: { model: "schedule.name" } }
-      %div{ ng: { show: "submitted && schedule_form.$pristine" } }
-        .error{ ng: { show: "(schedule_form.name.$error.required)" } }
+      %input.fullwidth{ "type" => 'text', "name" => 'name', "required" => true, "placeholder" => "{{ 'js.admin.order_cycles.schedules.schedule_name_placeholder' | t }}", "ng-model" => "schedule.name" }
+      %div{ "ng-show" => "submitted && schedule_form.$pristine" }
+        .error{ "ng-show" => "(schedule_form.name.$error.required)" }
           {{ 'js.admin.order_cycles.schedules.name_required_error' | t }}
 
     .order-cycles-selector.text-center.margin-bottom-30
 
     .text-center
 
-      %input.button{ type: 'submit', value: "{{ 'js.admin.order_cycles.schedules.create_schedule' | t }}", ng: { hide: 'schedule.id' } }
-      %input.button{ type: 'submit', value: "{{ 'js.admin.order_cycles.schedules.update_schedule' | t }}", ng: { show: 'schedule.id' } }
-      %span{ ng: { show: 'schedule.id' } } or
-      %input.button.red{ type: 'button', value: "{{ 'js.admin.order_cycles.schedules.delete_schedule' | t }}", ng: { show: 'schedule.id', click: 'delete()'} }
-      %input.button{ type: 'button', value: "{{ 'actions.cancel' | t }}", ng: { click: 'close()' } }
+      %input.button{ "type" => 'submit', "value" => "{{ 'js.admin.order_cycles.schedules.create_schedule' | t }}", "ng-hide" => 'schedule.id' }
+      %input.button{ "type" => 'submit', "value" => "{{ 'js.admin.order_cycles.schedules.update_schedule' | t }}", "ng-show" => 'schedule.id' }
+      %span{ "ng-show" => 'schedule.id' } or
+      %input.button.red{ "type" => 'button', "value" => "{{ 'js.admin.order_cycles.schedules.delete_schedule' | t }}", "ng-show" => 'schedule.id', "ng-click" => 'delete()' }
+      %input.button{ "type" => 'button', "value" => "{{ 'actions.cancel' | t }}", "ng-click" => 'close()' }
```

Comment:
I see here that all attributes for the element were re-written.
It converts the clean symbol notation to string/rocket notation which isn't necessary.

But.. maybe we can just look away because we'd like to say bye bye Angular anyway..